scripts/postprocessing/coverage_analysis/activity_db_generator.py

```diff
@@ -7,6 +7,7 @@
 import sys
```

Comment:
We should probably add andres to check this and the other coverage analysis changes